incident_viewer_demo/src/LineChart/LineChart.jsx

```diff
@@ -735,21 +719,21 @@ class LineChart extends PureComponent {
       // Take the series data, filter it for only in the good and bad ranges, then map those dates to their download speeds.
       // Use resulting download speeds to bound shaded regions.
       series[0].results
-        .filter(entry => (entry.date.isSameOrAfter(goodIncidentSeries.start.clone()) && entry.date.isSameOrBefore(goodIncidentSeries.end.clone())))
+        .filter(entry => (entry.date.isSameOrAfter(incident.goodPeriodStart.clone()) && entry.date.isSameOrBefore(incident.goodPeriodEnd.clone())))
```

Comment:
Can you remove the clone() part (and everywhere else it appears). This was leftover code for the shading that if added, will be added later.

Reply:
Done!